packages/itwinui-css/src/utils/icon/icon-component.scss

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@use './icon' as *;
+
+@mixin iui-svg-icon {
+  --_iui-svg-size: #{map-get($iui-icon-sizes, m)};
+  --_iui-svg-fill: var(--iui-color-icon-muted);
+
+  display: inline-flex;
+  flex-shrink: 0;
```

Comment:
What are these two lines for?

Reply:
`flex-shrink: 0` is to ensure that the icon maintains its size when used in a flex container.

`inline-flex` is to make sure the element wrapping the svg only takes up the height of the svg. without it, you will find that the height increases by a few pixels and the vertical alignment gets messed up.